### ISSUE TITLE
[documentation] https://letsencrypt.readthedocs.io should redirects to https://certbot.eff.org/docs/

### ISSUE THREAD

**Issue by tdelmas** [certbot/certbot#6444]
Or at least displays a banner to says that the documentation is now hosted on https://certbot.eff.org/docs/ since #2845 is closed.

**Comment by bmw** [comment]
I agree that's a good idea. Do you know if readthedocs has a configuration option for this?

**Comment by iammyr** [comment]
Yes you can @tdelmas and @bmw see from https://docs.readthedocs.io/en/latest/user-defined-redirects.html 


> Page Redirects can redirect URLs outside Read the Docs platform just by defining the “To URL” as the absolute URL you want to redirect to.

I would do this myself but apparently you need to log into your readthedocs.org account. So one of the admins needs to do that.
